thirdparty/gflags/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(GFLAGS_IS_SUBPROJECT TRUE)
+set(HAVE_SYS_STAT_H 1)
+set(HAVE_INTTYPES_H 1)
+set(INTTYPES_FORMAT C99)
+set(BUILD_TESTING OFF)
+set(BUILD_SHARED_LIBS OFF)
+
+add_subdirectory(gflags EXCLUDE_FROM_ALL)
+set_target_properties(gflags_nothreads_static PROPERTIES FOLDER thirdparty)
+
+if(COMMAND openvino_developer_export_targets)
+    openvino_developer_export_targets(COMPONENT openvino_common TARGETS gflags)
+endif()
```

Comment:
just move to `thirdparty/CMakeLists.txt`

Reply:
Yes, already done 👍

Reply:
Done